### PR TITLE
Improve performance of LogView

### PR DESCRIPTION
**Pull request by brentpicasso**
* Improve performance of LogView window
* Prevent large volume of updates from noticeably slowing down app
* Adjust max size of buffer to accommodate mobile or desktop platforms to reflect differences in performance
* Add copy log to clipboard for convenience 
* Refactoring so main.py isn't aware of logfile polling
* Makes Poll log label responsive to touch so it toggles checkbox state
* Documentation and other refactorings

### CODE REVIEW

**Comment by brentpicasso** [comment]
Resolves #930 #939

**Comment by brentpicasso** [comment]
test script for generating high volume of log messages:


i=0
setTickRate(30)

function onTick() 
  println('The quick brown fox jumped over ' ..i ..' lazy dogs')
  i=i+1

end

File: autosportlabs/racecapture/views/configuration/rcp/scriptview.kv [hunk 1]
```diff
@@ -43,41 +50,46 @@
...
+            BoxLayout:
                 size_hint_x: 0.03                                               	
             LabelIconButton:
                 size_hint_x: None
-                width: dp(100)
+                width: dp(90)
                 title: "Run"
                 icon: "\357\200\241"
-                on_press: root.runScript()
+                on_press: root.run_script()
```
Comment (stieg) [inline autosportlabs/racecapture/views/configuration/rcp/scriptview.kv]: Missing file newline.

**Comment by stieg** [comment]
Code looks good.  I left my unit at the office so I can't test tonight.  I'll give it a shot tomorrow with as script I knew caused issues with log reading....  :shipit:

**Pull request merged**